### PR TITLE
fix: Removed references to Atala in docs

### PR DESCRIPTION
**Pull request by todorkoleviohk**
Under [ATL-6478](https://input-output.atlassian.net/browse/ATL-6478) and according to the replacement table, references to Atala were renamed to conform with brand position.

The changes are mostly to "readme" and other md files, as changes to the code require in-depth assessment of impact by an expert of this artefact.

[ATL-6478]: https://input-output.atlassian.net/browse/ATL-6478?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: static/img/hosted-instances-overview-4.svg
Comment (yshyn-iohk) [inline static/img/hosted-instances-overview-4.svg:1]: We don't have `Cloud Agent Enterprise` anymore.
I think the square `Cloud Agent Enterprise` must be removed, and the arrows from  `Manage`,` Onboard,` and `Authenticate` must lead to the `Cloud Agent`.
CC: @bvoiturier, @milosh86

**Review by yshyn-iohk — APPROVED** [review]
Everything is fine, svg file should be slightly modified.
Let's commit it as it is and change the file after feedback from @bvoiturier and @milosh86 .

File: documentation/docs/getting-started.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 1
...
+Hyperledger Identus is a comprehensive toolset that provides the necessary infrastructure for decentralized identity systems. It operates as a layer-2 blockchain solution, using the Distributed Ledger as a [verifiable data registry (VDR)](/docs/concepts/glossary#verifiable-data-registry). This innovative toolset facilitates the creation and management of decentralized identifiers (DIDs) and supports the issuance, updating, and revocation of verifiable credentials. With this technology, users can securely and efficiently manage their digital identities while ensuring the integrity and authenticity of their credentials.
```
Comment (petevielhaber) [inline documentation/docs/getting-started.md:7]:
```suggestion
Hyperledger Identus is a comprehensive toolset that provides the necessary infrastructure for decentralized identity systems. It operates as a layer-2 blockchain solution, using a distributed ledger as a [verifiable data registry (VDR)](/docs/concepts/glossary#verifiable-data-registry). This innovative toolset facilitates the creation and management of decentralized identifiers (DIDs) and supports the issuance, updating, and revocation of verifiable credentials. With this technology, users can securely and efficiently manage their digital identities while ensuring the integrity and authenticity of their credentials.
```

File: documentation/docs/identus/cloud-agent/overview.md [hunk 2]
```diff
@@ -47,11 +47,11 @@ This document provides an overview of the Cloud Agent feature set. This document
...
+| --- | :--: |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `did:PRISM` | :white_check_mark: | W3C [Decentralized Identifiers (DIDs) v1.0](https://www.w3.org/TR/did-core/) compliant [PRISM DID Method](https://github.com/input-output-hk/PRISM-did-method-spec/blob/main/w3c-spec/PRISM-method.md) specification. The Distributed Ledger is the default [Verifiable Data Registry (VDR)](/docs/concepts/glossary#verifiable-data-registry) to anchor DIDs. PRISM DID Method is used for [Verifiable Credentials](/docs/concepts/glossary#verifiable-credential) and [Verifiable Presentations](/docs/concepts/glossary#verifiable-presentation). |
```
Comment (petevielhaber) [inline documentation/docs/identus/cloud-agent/overview.md:52]:
```suggestion
| `did:PRISM` | :white_check_mark: | W3C [Decentralized Identifiers (DIDs) v1.0](https://www.w3.org/TR/did-core/) compliant [PRISM DID Method](https://github.com/input-output-hk/PRISM-did-method-spec/blob/main/w3c-spec/PRISM-method.md) specification. A distributed ledger is the default [Verifiable Data Registry (VDR)](/docs/concepts/glossary#verifiable-data-registry) to anchor DIDs. PRISM DID Method is used for [Verifiable Credentials](/docs/concepts/glossary#verifiable-credential) and [Verifiable Presentations](/docs/concepts/glossary#verifiable-presentation). |
```

File: documentation/docs/identus/prism-node.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # PRISM Node
-The PRISM Node acts as a second-layer node for the Cardano blockchain. It functions as a [verifiable data registry](/docs/concepts/glossary#verifiable-data-registry), providing a secure and reliable way to store and manage data.
+The PRISM Node acts as a second-layer node for the Distributed Ledger. It functions as a [verifiable data registry](/docs/concepts/glossary#verifiable-data-registry), providing a secure and reliable way to store and manage data.
```
Comment (petevielhaber) [inline documentation/docs/identus/prism-node.md:2]:
```suggestion
The PRISM Node acts as a second-layer node for the distributed ledger. It functions as a [verifiable data registry](/docs/concepts/glossary#verifiable-data-registry), providing a secure and reliable way to store and manage data.
```

File: documentation/docs/identus/prism-node.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+The PRISM Node acts as a second-layer node for the distributed ledger. It functions as a [verifiable data registry](/docs/concepts/glossary#verifiable-data-registry), providing a secure and reliable way to store and manage data.
 
-PRISM Node's primary purpose is to provide a secure and trustworthy platform for storing and managing data on the Cardano blockchain. By leveraging the blockchain's security and decentralization, the PRISM Node stores and retrieves data in a secure and immutable manner. All operations are independently verified and authenticated using cryptographic signatures and other security measures, so all data is accurate and trustworthy.
+PRISM Node's primary purpose is to provide a secure and trustworthy platform for storing and managing data on the Distributed Ledger. By leveraging the blockchain's security and decentralization, the PRISM Node stores and retrieves data in a secure and immutable manner. All operations are independently verified and authenticated using cryptographic signatures and other security measures, so all data is accurate and trustworthy.
```
Comment (petevielhaber) [inline documentation/docs/identus/prism-node.md:4]:
```suggestion
PRISM Node's primary purpose is to provide a secure and trustworthy platform for storing and managing data on the distributed ledger. By leveraging the blockchain's security and decentralization, the PRISM Node stores and retrieves data in a secure and immutable manner. All operations are independently verified and authenticated using cryptographic signatures and other security measures, so all data is accurate and trustworthy.
```

**Comment by yshyn-iohk** [comment]
https://github.com/hyperledger/identus-docs/pull/135

**Pull request closed without merging**